cli/tests/snapshot/snapshots/snapshot__pprint-ast_stdout_simple_record.ncl.snap

```diff
@@ -4,12 +4,15 @@ expression: out
 ---
 {
   a | Number | default
-    = 1,
+    =
+    1,
```

Comment:
In principle, yes; but this snapshot still looks weird because it splits after the `=` but keeps the metadata annotations on the same line. Will investigate.

Reply:
Changed in [5d4e9ea](https://github.com/tweag/nickel/pull/1532/commits/5d4e9ea74f3e94474f1e901cfcd92fe89a40569f)